config/environments/production.rb

```diff
@@ -64,8 +64,6 @@
   # Set this to true and configure the email server for immediate delivery to raise delivery errors.
   # config.action_mailer.raise_delivery_errors = false
   config.action_mailer.delivery_method = :smtp
```

Comment:
No, it won't be a problem unless someone tries to send email. When they do, it's their problem :smile: